.kokoro/setup-vars.sh

```diff
@@ -20,4 +20,7 @@
 export AGENT_ID="be66a5a2-4602-4f8c-a3f3-046896904362"
 export TEST_ID="9ebddb66-c49e-455b-bacc-24c0538ece01"
 
+export TRAINING_PHRASE_AGENT_ID="5d23f659-cd71-43e9-8fb2-b69cd9896370"
```

Comment:
I'm curious - instead of modifying files in the `.kokoro` directory, could we do this in the sample test code?  Something like:

```
const trainingPhraseAgentId = process.env.TRAINING_PHRASE_AGENT_ID || '5d23f659-cd71-43e9-8fb2-b69cd9896370';
```

I only ask because these are going to be harder and harder to keep track of as we evaluate non-kokoro solutions, and uniformity here is delightful.